File: ViewletCreatorIOS/ViewletCreator/Classes/binder/ViewletBinder.swift
//
//  ViewletBinder.swift
//  Viewlet creator Pod
//
//  Library binder: default protocol
//  A protocol which can be implemented to do custom viewlet binding
//

import UIKit

public protocol ViewletBinder {
    
    func onBind(refId: String, view: UIView)
    
}

File: ViewletCreatorAndroid/ViewletCreatorLib/src/main/java/com/crescentflare/viewletcreator/binder/ViewletBinder.java
package com.crescentflare.viewletcreator.binder;

import android.view.View;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

/**
 * Viewlet creator binder: default interface
 * An interface which can be implemented to do custom viewlet binding
 */
@SuppressWarnings({"unused", "WeakerAccess"})
public interface ViewletBinder
{
    void onBind(@NotNull String refId, @Nullable View view);
}
